package.json

```diff
@@ -27,9 +27,9 @@
   ],
   "devDependencies": {
     "@netflix/eslint-config": "^3.0.0",
-    "eslint": "^9.0.0",
-    "puppeteer": "^22.0.0",
-    "tap-parser": "^15.3.1"
+    "eslint": "^9.8.0",
+    "puppeteer": "^23.0.2",
+    "tap-parser": "^18.0.0"
```

Comment:
Hrm, they sorta bundled all the `tapjs` stuff into a single changelog. It’s much harder to tell exactly what changed related to the `tap-parser` library. Anyways, it should still be working with TAP 14 output, so that should still interop with `x-test`.